### PR TITLE
Try again

### PR DESCRIPTION
**Pull request by Frankonly**
#42

### CODE REVIEW

File: blockchain/action/execution_test.go [hunk 1]
```diff
@@ -72,3 +72,32 @@ func TestExecutionSerializeDeserialize(t *testing.T) {
 	require.Equal(ex.Hash(), newex.Hash())
 	require.Equal(ex.TotalSize(), newex.TotalSize())
 }
+
+func TestExecutionToJSONFromJSON(t *testing.T) {
+	require := require.New(t)
+	executor, err := iotxaddress.NewAddress(true, chainid)
+	require.Nil(err)
```
Comment (zjshen14) [inline blockchain/action/execution_test.go]: Prefer to use`require.NoError`

Same for the error assertions bellow

Comment (Frankonly) [reply blockchain/action/execution_test.go]: `require.Nil` is used in other test functions writed by others, so I use it here. Should I change all of them?

**Comment by Zhen-iotex** [comment]
In JSON format conversion test, could you also sign the action, it seems the NewVoteFromJSON does not include the pbVote into the actual vote.

**Comment by Frankonly** [comment]
Finally all checks passed.

**Comment by zjshen14** [comment]
@Frankonly congrats on ur first blood! Looking forward to more PRs!

**Comment by Frankonly** [comment]
@zjshen14 Thanks~

**Pull request merged**